airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -725,7 +657,7 @@ def set_task_instance_note(
         ti.task_instance_note.content = new_note
         ti.task_instance_note.user_id = current_user_id
     session.commit()
-    return task_instance_schema.dump((ti, sla_miss))
+    return task_instance_schema.dump((ti,))
```

Comment:
probably this should not want to take a tuple anymore.

Reply:
I tried that but it blew up in the `task_instance_schema.dump(ti)` call because "TaskInstance is not subscriptable"  I just added a commit which handles it, let me know what you think